sabnzbd/__init__.py

```diff
@@ -1087,10 +1087,10 @@ def test_ipv6():
 
     try:
         af, socktype, proto, canonname, sa = info[0]
-        sock = socket.socket(af, socktype, proto)
-        sock.settimeout(2)  # 2 second timeout
-        sock.connect(sa[0:2])
-        sock.close()
+        with socket.socket(af, socktype, proto) as sock:
+            sock.settimeout(2)  # 2 second timeout
+            sock.connect(sa[0:2])
+            sock.close()
```

Comment:
This line needs to be removed, since you use the `with`.

Reply:
Thanks. Solved with additional commit.